cli/src/config-schema.json

```diff
@@ -164,7 +164,8 @@
                     "description": "Conflict marker style to use when materializing conflicts in the working copy",
                     "enum": [
                         "diff",
-                        "snapshot"
+                        "snapshot",
+                        "git-diff3"
                     ],
```

Comment:
(For discussion) I checked, and the `diff3` format is not meant for being perfectly decodable in the sense of #3975. Git and `diff3` don't do anything special if the text contains conflict markers, and generate text that cannot be parsed unambiguously.

I wonder whether, in these cases we should fall back to other conflict styles or fail to materialize (it could, for example, output to something similar as what `jj` does when symlinks conflict). We *probably* should do the same thing in this case as in the case when there are too many sides.

It's fine to just do a fallback for now and maybe implement something else later (e.g. a "git-strict" style that fails on error), as long as we have a sense that it won't be a hopeless mess.

Reply:
Turns out Git sort of supports longer conflict markers, see https://github.com/martinvonz/jj/pull/4969#discussion_r1859088488.